bitrise.yml

```diff
@@ -125,23 +93,16 @@ workflows:
         inputs:
         - content: |-
             #!/bin/bash
-            set -e
-            set -x
-
-            # Install dependencies
-            go get -u github.com/tools/godep
+            set -ex
 
             # Check for unhandled errors
             go get -u github.com/kisielk/errcheck
 
             # Go lint
             go get -u github.com/golang/lint/golint
 
-            # Check for unused constants, variables, functions and types
-            go get honnef.co/go/tools/cmd/unused
-
-            # Search for what could be simplified
-            go get honnef.co/go/tools/cmd/gosimple
+            # go-utils/command
+            go get -u github.com/bitrise-io/go-utils/command
```

Comment:
are you sure? go-utils/command seems unnecessary, but honnef.co/go/tools/cmd/unused might be useful

that said, I'd remove these go gets and instead do it before the tool would be used